packages/raystack/typings.d.ts

```diff
@@ -1,6 +1,22 @@
-declare module "*.module.css";
+declare module "*.module.css" {
+  const classes: { [key: string]: string };
+  export default classes;
+}
 
 declare module "*.svg" {
   const content: React.FunctionComponent<React.SVGAttributes<SVGElement>>;
   export default content;
 }
+
+export declare module "@tanstack/table-core" {
```

Comment:
These two exports were defined in `typing.d.ts`. Just moved theme in this file `typings.d.ts`.